src/components/Confirm/PdfPreview.js

```diff
@@ -0,0 +1,61 @@
+import { h, Component } from 'preact'
+import PDFObject from 'pdfobject'
+import {preventDefaultOnClick} from '../utils'
+
+const FallbackView = ({blob}) => {
+  const downloadPDF = (e) => {
+    window.navigator.msSaveOrOpenBlob(blob, 'document.pdf')
+  }
+  return (
+    <div>
+      <a href='#' onClick={preventDefaultOnClick(downloadPDF)}><div id='pdfIcon'></div></a>
+    </div>
+  )
+}
+
+class PDFPreview extends Component {
+  options = {
+    'width': '92%',
+    'height': '290px',
+    'max-height': '70vh',
+    'border': 0,
+    fallbackLink: "<a href='[url]'><div id='pdfIcon'></div></a>"
+  }
+
+  embedPDF(previewUrl){
+    PDFObject.embed(previewUrl, '#pdfContainer', this.options);
+  }
+
+  componentDidMount() {
+    const { previewUrl } = this.props;
+    this.embedPDF(previewUrl)
+  }
+  shouldComponentUpdate (nextProps, nextState) {
+    return false;
+  }
+  componentWillReceiveProps({previewUrl}) {
+    if (this.props.pdfPreview !== previewUrl) this.embedPDF(previewUrl)
+  }
+  render() {
+    return <div id='pdfContainer' />;
```

Comment:
No it wouldn't - CSS modules generate their output based on the file path + file name

Plus it would be a really weird hack to do that :P

Just put a `let i = 0;` outside of the class, and then add a `this.id = 'pdfContainer' + (i++)` to the contructor